### DISCUSSION TITLE
I only see a white screen on my iOS build

### DISCUSSION THREAD

**Discussion by Vinnie881** [quasarframework/quasar#16464] [CLI - Capacitor / Cordova modes]
Everything is working fine when I run in dev on my iOS, but when I run a real build I just get a white screen. 

I was under the impression it should work, but I have not changed any defaults for ip or anything else.  It seems like the iphone wants a server running somewhere.  Is there something I have setup wrong or a reason I get a white screen only on my builds.   Dev launches my iOS perfect and all works the same as on a browser, just the build gets me nodda:(

**Comment by Munaro** [comment]
Same problem here.

**Comment by k-sym** [comment]
Same here. Anyone?

**Comment by itsHadyy** [comment]
Same here

**Reply by k-sym** [reply]
Installing cordova-plugin-wkwebview-engine worked for me.

**Reply by itsHadyy** [reply]
I'll try that, hope it works. Thank you.
